pyiode/iode_database/variables_database.pyx

```diff
@@ -891,6 +900,7 @@ cdef class Variables(_AbstractDatabase):
 
         Parameters
         ----------
+        # GDM> use Enum or str?
```

Comment:
I haven't had the time to fully understand what this method does when I added it to Cython. I'll write a better documentation as soons as possible.